test/testFixtures/settings.json

```diff
@@ -17,8 +17,8 @@
   "ansible.python.activationScript": "",
   "ansible.python.interpreterPath": "python3",
   "ansibleServer.trace.server": "verbose",
-  "ansible.lightspeed.enabled": false,
-  "ansible.lightspeed.suggestions.enabled": false,
+  "ansible.lightspeed.enabled": true,
```

Comment:
Unless I set these two settings to `true`, the Ansible panel won't open at the content matches text execution and it fails. It shouldn't fail because we explicitly set these two settings to `true` in the test code. There might be some timing or test-specific issues. I will take at this issue further before sending this PR for review.